src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -568,33 +568,25 @@ private static IsolatedStorageFile GetStore(IsolatedStorageScope scope)
         public static IsolatedStorageFile GetStore(IsolatedStorageScope scope, Type applicationEvidenceType)
         {
             // Scope MUST be Application
-
-            // https://github.com/dotnet/corefx/issues/10935
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_CAS); // https://github.com/dotnet/corefx/issues/10935
         }
 
         public static IsolatedStorageFile GetStore(IsolatedStorageScope scope, object applicationIdentity)
         {
             // Scope MUST be Application
-
-            // https://github.com/dotnet/corefx/issues/10935
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_CAS); // https://github.com/dotnet/corefx/issues/10935
```

Comment:
The `applicationEvidenceType` is used to specify which type from the AppDomain Evidence to use to generate the folder structure. It has nothing to do with CAS, just storage bucket identity. In this case I _could_ allow for the types we're able to emulate and throw for others- I didn't do it at the time as we were going to pull the types that would be necessary to do so (notably StrongName).